teku/src/main/java/tech/pegasys/teku/cli/deposit/DepositGenerateAndRegisterCommand.java

```diff
@@ -96,8 +97,34 @@ public void run() {
   @NotNull
   private Function<ValidatorKeys, SafeFuture<TransactionReceipt>> registerValidator(
       final RegisterAction registerAction) {
-    return validatorKey ->
-        registerAction.sendDeposit(
-            validatorKey.getValidatorKey(), validatorKey.getWithdrawalKey().getPublicKey());
+    return validatorKey -> {
+      final BLSKeyPair validatorKeyPair = validatorKey.getValidatorKey();
+      final BLSKeyPair withdrawalKeyPair = validatorKey.getWithdrawalKey();
+      if (displayConfirmation) {
```

Comment:
Should be a different flag I think, the displayConfirmation flag seems like different setting to suppressing the output

Reply:
flag updated.